blueprints/vendor-shim/files/vendor/__name__-shim.js

```diff
@@ -0,0 +1,9 @@
+(function() {
```

Comment:
Unsure if related, but we should generally not generate individual files in `vendor/` (they should be in subdirectories).

https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/ember-app.js#L1372